update-gke-deployment/action.yaml

```diff
@@ -51,3 +51,9 @@ runs:
       shell: bash
       run: |-
         kubectl rollout status deployment "${{ inputs.deployment-name }}-${{ inputs.deployment-environment }}" --timeout="${{ inputs.deployment-rollout-timeout }}" --namespace="${{ inputs.deployment-namespace }}"
+    - name: Deployment failure
+      shell: bash
+      if: ${{ inputs.deployment-rollout-timeout }} == '10m'
+      run: |-
+        print "Failure to deploy ${{ inputs.deployment-name }}-${{ inputs.deployment-environment }}"
+        kubectl describe deploy "${{ inputs.deployment-name }}-${{ inputs.deployment-environment }}"
```

Comment:
Nice touch @ericblancas, we might want to move this about the print statement above just because this describe output is very large and I think the deployment failure message will get lost.